Installers/default.build

```diff
@@ -106,10 +107,11 @@
          </fileset>
       </copy>
       <copy todir="root/Library/Frameworks/MonoGame.framework/v${buildNumber}/Assemblies/DesktopGL">
-         <fileset basedir="../MonoGame.Framework/bin/DesktopGL/AnyCPU/Release">
+         <fileset basedir="../MonoGame.Framework/bin/Linux/AnyCPU/Release">
```

Comment:
also if you look at https://github.com/dellis1972/MonoGame/blob/addinfix/Installers/default.build#L169 that is exactly what the linux installer does atm